src/main/resources/application.properties

```diff
@@ -1,7 +1,9 @@
 server.port=8081
 spring.h2.console.enable=true
-spring.datasource.url=jdbc:h2:mem.test
+spring.datasource.url=jdbc:h2:mem:db_test
```

Comment:
The database to use is Postgresql. The card https://trello.com/c/iGFbaf25/18-creacion-del-proyecto-base tells you what to install and what properties are used to configure it in java

Reply:
I'm with a Windows machine, try to install Postgres and it gives an error COMSPEC which is an environment variable pointing to cmd.exe.
In the project I am using H2 to be able to continue.